ui/src/assets/schema/rules/sidecars/asl.yaml

```diff
@@ -120,7 +141,10 @@ MRIASLPcaslSpecific:
   fields:
     PCASLType:
       level: recommended
-      level_addendum: if `ArterialSpinLabelingType` is `"PCASL"`
+      level_addendum: recommended if `ArterialSpinLabelingType` is `PCASL`
```

Comment:
`PCASLType` is already recommended, does the addendum change anything?

Reply:
@dlevitas

Reply:
The level_addendum specifies conditions where the metadata might become required (or recommended), based on other metadata values